eventually/authentication/models.py

```diff
@@ -0,0 +1,161 @@
+        :return: user object or None if a user with such ID does not exist
+        """
+        try:
+            return CustomUser.objects.get(id=user_id)
+        except CustomUser.DoesNotExist:
+            return None
+
+    @staticmethod
+    def delete_by_id(user_id):
+        """
+        :param user_id: an id of a user to be deleted
+        :return: True if object existed in the db and was removed or False if it didn't exist
+        """
+        try:
+            user = CustomUser.get_by_id(user_id)
+            user.delete()
+            return True
+        except AttributeError:
+            return False
+
+    @staticmethod
+    def create(first_name, middle_name, last_name, email, password):
+        """
+        :param first_name: first name of a user
+        :param middle_name: middle name of a user
+        :param last_name: last name of a user
+        :param email: email of a user
+        :param password: password of a user
+        :return: a new user object which is also written into the DB
+        """
+        user = CustomUser(first_name=first_name,
+                          last_name=last_name,
+                          middle_name=middle_name,
+                          email=email,
+                          password=password)
+        user.save()
+        return user
+
+    #'FIXME: remove 4 methods below? Do we need them?'
+    def get_full_name(self):
+        """
+        :return: first, middle and last name of the created user
+        """
+
+        return '{} {} {}'.format(self.first_name, self.middle_name, self.last_name)
+
+    def get_email(self):
+        """
+        :return: email of the user.
+        """
+        return self.email
+
+    def get_updated_at(self):
+        """
+        :return: returns last update date in form of a timestamp
+        """
+        date = format(self.updated_at, u'U')
+        return date
+
+    def get_created_at(self):
+        """
+        :return: creation date of a user record in form of a timestamp
+        """
+        return format(self.created_at, u'U')
+
+    #'FIXME: which of these to remove? to_dict or myattr_to_dict ?'
+    def to_dict(self):
+        """
+        :return: Returns an object in form of a dictionary. Methods inherited from the parent object are also shown.
+        """
+        return self.__dict__
+
+    def myattr_to_dict(self):
+        """
+        :return: manually-defined attributes of the object. Inherited attributes aren't shown.
+        """
+        return {
+                'id': self.id,
+                'first_name': self.first_name,
+                'middle_name': self.middle_name,
+                'last_name': self.last_name,
+                'email': self.email,
+                'password': self.password,
+                'created_at': format(self.created_at, u'U'),
+                'updated_at': format(self.updated_at, u'U')
+                }
+
+    def update(self, **kwargs):
```

Comment:
This way user can input any arg if u used *kwargs

Reply:
More simply way to use only `if` check for each possible argument without using kwargs.

Reply:
Yes, but this way he'll have to enter all of the attributes. What if he wants to change only 1 or 2 of them?